how-to/install-system-packages.rst

```diff
@@ -43,4 +41,4 @@ having to rebuild the entire project, jump into a new container running
 
      docker-compose run --rm web bash
 
-This container will disappear (``--rm``) when you exit.
+``--rm`` will make the container disappear when you exit.
```

Comment:
Was better previously, please revert this change.

Reply:
Done